src/jsd/actuator.h

```diff
@@ -67,7 +69,41 @@ class Actuator : public JsdDeviceBase
   static bool        IsJsdFaultCodePresent(const DeviceState& state);
   static double      GetActualPosition(const DeviceState& state);
 
+  struct ActuatorParams {
+    std::string actuator_type_str;
```

Comment:
This did not get populated when doing the merge. I will populate it and add `ctrl_gs_mode_str` to the parameters too.